storybook/src/data/documents.data.js

```diff
@@ -0,0 +1,66 @@
+import uuid from 'uuid/v4';
+import image from './nuxeo-logo.jpg';
+
+export default class Document {
+  // class methods
+  constructor(type = '', properties = {}, facets = [], permissions = [], isTrashed = false, isFavorite = false) {
+    this.type = type;
+    this.properties = properties;
+    this.facets = facets;
+    this.permissions = permissions;
+    this.isTrashed = isTrashed;
+    this.isFavorite = isFavorite;
+  }
+
+  trash() {
+    this.isTrashed = true;
+    return this;
+  }
+
+  favorite() {
+    this.isFavorite = true;
+    return this;
+  }
+
+  addType(documentType) {
```

Comment:
I think we could avoid using verbs here and instead the name of the property we're setting, particularly because we are not adding but setting?
```suggestion
type(documentType) {
```

Reply:
I would suggest to keep verbs since it is a more common pattern for builders and it is also important if you want something like `addPermission` and `removePermission` or `setPermissions`.